src/lib/swapper/swappers/ThorchainSwapper/getThorTradeQuote/getTradeQuote.test.ts

```diff
@@ -65,12 +65,12 @@ describe('getTradeQuote', () => {
           return Promise.resolve({
             data: {
               dust_threshold: '10000',
-              expected_amount_out: '261454522054192',
+              expected_amount_out: '10232161',
               expiry: 1681132269,
               fees: {
                 affiliate: '0',
-                asset: 'ETH.FOX-0XC770EEFAD204B5180DF6A14EE197D99D808EE52D',
```

Comment:
Our mocked quote response here was for an ETH to FOX trade, but the tests are expecting a FOX to ETH trade. As we are now using this data to compute the quote details, this had to be updated (and expected test values changed accordingly).